src/screens/Authentication/phonenumberinput.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useRef } from 'react';
+import { SafeAreaView, StyleSheet, Text, TouchableHighlight } from 'react-native';
+import PhoneInput from 'react-native-phone-input';
+
+const PhoneNumberInput = () => {
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const phoneInput = useRef<PhoneInput>(null);
+
+  const handleSubmit = () => {
+    const isValid = phoneInput.current?.isValidNumber(phoneNumber);
+    if (isValid) {
+      console.log('SUBMITTED! ', phoneNumber);
+    } else {
+      console.log('INVALID NUMBER.');
+    }
+  };
+  return (
+    <SafeAreaView>
+      <PhoneInput
+        style={styles.phoneInput}
+        initialValue={phoneNumber}
+        initialCountry="us"
+        onChangeText={(text) => {
+          setPhoneNumber(text);
+        }}
+        withShadow
+        autoFocus
+      />
+      <TouchableHighlight onPress={handleSubmit}>
+        <Text>Submit</Text>
+      </TouchableHighlight>
+    </SafeAreaView>
+  );
+};
+const styles = StyleSheet.create({
```

Comment:
I noticed an inconsistency with screens/components being styled using StyleSheets in file vs imported from another file. Based on other comments, you sh prob move this to a separate styles file :)